### PR TITLE
Update Cell Profiler workflow

### PR DESCRIPTION
**Pull request by beatrizserrano**

### CODE REVIEW

File: topics/imaging/tutorials/tutorial-CP/workflows/main_workflow.ga [hunk 1]
```diff
@@ -1,1143 +1,1138 @@
...
-      "annotation": "",
-      "content_id": "toolshed.g2.bx.psu.edu/repos/bgruening/cp_cellprofiler/cp_cellprofiler/3.1.9",
-      "type": "tool"
-    }
-  },
-  "annotation": "Introduction to image analysis using Galaxy",
-  "a_galaxy_workflow": "true"
-}
\ No newline at end of file
+    },
```
Comment (bgruening) [inline topics/imaging/tutorials/tutorial-CP/workflows/main_workflow.ga:1131]: "tags": [
    "CellProfiler",
    "imaging"
  ],

I think....

Comment (beatrizserrano) [reply topics/imaging/tutorials/tutorial-CP/workflows/main_workflow.ga:1131]: this is the workflow as I export it from Galaxy, did I do something wrong at an earlier stage?

**Comment by bgruening** [comment]
Don't know but the new one is valid and working.

**Review by bgruening — APPROVED** [review]

**Comment by hexylena** [comment]
@bgruening @beatrizserrano this was merged over a failing test, so unfortunately now new PRs are also failing. Would you mind to fix the annotation tag please?

```
Linting topics/imaging/tutorials/tutorial-CP/workflows/main_workflow.ga

	The 'annotation' attribute is missing or incorrect. Please add: "annotation": "<title of tutorial>"
```

**Comment by hexylena** [comment]
Nevermind, Saskia tells me she fixed it in https://github.com/galaxyproject/training-material/pull/2114

**Comment by beatrizserrano** [comment]
Oh no, sorry! Thanks @shiltemann  @hexylena :)

**Pull request merged**